fv3core/stencils/neg_adj3.py

```diff
@@ -1,177 +1,151 @@
 import gt4py.gtscript as gtscript
-import numpy as np
 from gt4py.gtscript import BACKWARD, FORWARD, PARALLEL, computation, interval
 
 import fv3core._config as spec
 import fv3core.utils.global_constants as constants
 import fv3core.utils.gt4py_utils as utils
 from fv3core.decorators import gtstencil
+from fv3core.utils.typing import FloatField, FloatFieldIJ
 
 
 sd = utils.sd
 ZVIR = constants.RVGAS / constants.RDGAS - 1.0
 
 
 @gtscript.function
-def fix_negative_ice(qv, qi, qs, qg, qr, ql, pt, lcpk, icpk, dq):
-    qsum = qi + qs
+def fix_negative_ice(qvapor, qice, qsnow, qgraupel, qrain, qliquid, pt, lcpk, icpk, dq):
```

Comment:
Type annotations are nice on `gtscript.function`'s too, but not required.

Reply:
Would you object to me not including them this PR? I think we'll be revisiting this file a bit in the future

Reply:
No objection at all!